src/Poe/Entity.cs

```diff
@@ -14,6 +16,8 @@ public sealed class Entity : RemoteMemoryObject, IEntity
         public long Id => (long)M.ReadInt(Address + 0x40) << 32 ^ Address;
         public int InventoryId => M.ReadInt(Address + 0x58);
 
+        public int Distance => GetDistance();
```

Comment:
wait...why do we have two functions???? Remove other one and keep this one.
In the other class u can do the following
Distance = InternalEntity.Distance
but I don't think we need 2 functions that is doing same thing...